README.md

```diff
@@ -1,68 +1,4 @@
 # Verify Matching Service Test Tool
 
-A [local matching service](http://alphagov.github.io/rp-onboarding-tech-docs/pages/ms/msWorks.html) allows you to find a match between a user’s verified identity and a record in your organisation's database(s).
+This repository has been moved [here](https://github.com/alphagov/verify-matching-service-adapter/verify-matching-service-test-tool).
```

Comment:
I think the URL is https://github.com/alphagov/verify-matching-service-adapter/tree/master/verify-matching-service-test-tool.